src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -5090,17 +5133,18 @@ void ExportMusicXml::hairpin(Hairpin const* const hp, staff_idx_t staff, const F
     }
 
     directionTag(m_xml, m_attr, hp);
-    if (hp->tick() == tick) {
-        writeHairpinText(m_xml, hp, hp->tick() == tick);
+    bool hpTick = hp->tick() == tick;
```

Comment:
```suggestion
    const bool hpTick = hp->tick() == tick;
```

Always good to see that it will not / should not change.

Reply:
sure, done